protostar/cheatable_starknet/callable_hint_locals/mock_call_hint_local.py

```diff
@@ -0,0 +1,26 @@
+from typing import Callable
+
+from protostar.cheatable_starknet.controllers.contracts import ContractsController
+from protostar.starknet import Address, Selector
+
+from .callable_hint_local import CallableHintLocal
+
+
+class MockCallHintLocal(CallableHintLocal):
+    def __init__(self, controller: ContractsController) -> None:
+        super().__init__()
```

Comment:
This call does not do anything, `CallableHintLocal` is an ABC

Reply:
It doesn't mean that `CallableHintLocal` can't have a constructor.

Reply:
But it doesn't, so why call it?

Reply:
To make the code less surprising. 

@MaksymilianDemitraszek WDYT?

Reply:
I agree with @kasperski95 here, If we would introduce some logic in the costructor in the future, It will prevent us from a weird silent bug

Reply:
Okay i feel convinced, resolving this